src/arena.c

```diff
@@ -39,6 +40,8 @@ const uint64_t h_steps[SMOOTHSTEP_NSTEPS] = {
 #undef STEP
 };
 
+static div_info_t arena_binind_div_info[NBINS];
```

Comment:
maybe create a beginner-friendly task to make this compile time const?

Reply:
I figure it doesn't matter so much in a world of dynamic size classes, which we expect to come shortly after this.